README.md

```diff
@@ -37,14 +37,14 @@ The former group deals with higher-level strategic and administrative decisions,
 The election process will involve the following phases.
 The dates are tentative and are subject to change depending on the community engagement during the nomination and voting phases.
 
-| Election phases                                                       | Tentative dates               |
-| --------------------------------------------------------------------- | ----------------------------- |
-| Nomination period                                                     | June 14–July 5                |
-| Nominee application review (by election committee and NumFOCUS board) | July 5–July 12                |
-| Notify nominees and get confirmation                                  | July 12–July 26               |
-| Voting period                                                         | July 26–Aug 9                 |
-| Review election results (by NumFOCUS board)                           | Aug 9–Aug 23                  |
-| Publish results announcing new board members                          | Late August / Early September |
+| Election phases                                                       | Tentative dates                |
+| --------------------------------------------------------------------- | ------------------------------ |
+| Nomination period                                                     | June 14–July 5                 |
+| Nominee application review (by election committee and NumFOCUS board) | July 5–July 12                 |
+| Notify nominees and get confirmation                                  | Aug 12–Aug 21                  |
+| Voting period                                                         | Aug 21– Sep 9                  |
+| Review election results (by NumFOCUS board)                           | Sep 9–Sep 23                   |
+| Publish results announcing new board members                          | Late September / Early October |
```

Comment:
```suggestion
| Notify nominees and get confirmation                                  | July 12–Sep 23                  |
| Voting period                                                         | Sep 23– Oct 7                  |
| Review election results (by NumFOCUS board)                           | Oct 7–Oct 23                   |
| Publish results announcing new board members                          | Late October / Early November |
```

Reply:
okay, do you need any support from the infrastructure team on the vote rollout?

Reply:
@tkoyama010 @kalyan678 - I think we can also mention the voter registration phase in the timeline. So, 
* Voter registration: 23rd Sep to 4th Oct 
* Voting period: 7th Oct - 25th Oct
* Review election results: 26th Oct - 8th Nov
* Publish results: mid Nov

Reply:
```suggestion
| Voter registration                                   | Sep 20–Oct 4                  |
| Voting period                                                         | Oct 7– Oct 25                  |
| Review election results (by NumFOCUS board)                           | Oct 26–Nov 8                   |
| Publish results announcing new board members                          | Middle Nov |
```